### PR TITLE
Move API documentation to API_v1.md

### PR DESCRIPTION
**Pull request by sugargoat**
### Motivation

The current API documentation is spread across two files (the README and API.md). This PR moves the usage from the README to API_v1.md, in anticipation of releasing version 2 of the API soon.

### In this PR
* Moves usage documentation from README to API.md
* Fixes typos and wording
* Adds usage for the get-raw-object methods

### CODE REVIEW

**Comment by christian-oudard** [comment]
Looks good! The only change that I notice is needed is to update the `API.md` link on line 86 of README.md to link to `API_v1.md`

**Review by christian-oudard — CHANGES_REQUESTED** [review]
Looks good! The only change that I notice is needed is to update the `API.md` link on line 86 of README.md to link to `API_v1.md`

**Review by jcape — APPROVED** [review]
Approved pending API.md link fix.

**Review by christian-oudard — APPROVED** [review]
Copying jcape, approved pending API_v1.md link.

File: README.md [hunk 1]
```diff
@@ -81,1145 +81,9 @@ MobileCoin Full Service is available under open-source licenses. Look for the [L
...
-  }
-}
-```
-
-| Required Param | Purpose                  | Requirements              |
-| :------------- | :----------------------- | :------------------------ |
-| `account_id`   | The account on which to perform this action  | Account must exist in the wallet  |
-| `txo_id`   | The ID of the Txo for which to verify the proof  | Txo must be a received Txo  |
-| `proof`   | The proof to verify  | The proof should be delivered by the sender of the Txo in question |
+For usage and API specification, see [API.md](./API.md).
```
Comment (jcape) [inline README.md:86]:
```suggestion
For usage and API specification, see [API_v1.md](./API_v1.md).
```

**Pull request merged**